### DISCUSSION TITLE
[Feature request] Being able to unstyle next/image component in order to make it compatible with post-processors like tailwind

### DISCUSSION THREAD

**Discussion by lveillard** [vercel/next.js#29340] [Ideas]
### Describe the feature you'd like to request

Hello there!
I'm completely unable to use this component:
https://github.com/vercel/next.js/blob/f2c266ba4d5bb3eeb8cdb674a0efd12a09af8374/packages/next/client/image.tsx

As it completely ignores any class given by tailwind and forces people using plain css to create child classes with` !important`   rules.

In my case i'm trying to add this utility classes to it:
`order-first lg:order-last mb-8 lg:mb-0 max-h-96 max-w-3/4 mx-auto`

In a way that the width and hieght of the photo are dynamic.

FMHO it's a bit weird to style a component. I understand in this case it will help 90% of devs but there is a lot of topics everywhere about this issue and it seems that one quick "unstyled" prop would make everyone happy

### Describe the solution you'd like

I would suggest to have a prop "unstyled" that would avoid generating the two parent divs and expose only the img with no style.. 
Of course that would remove some advantages but would empower devs to add the style they want to the image element.

### Describe alternatives you've considered

Maybe being able to pass props directly to the img container by imgStyle props?
But I think that would be like patching the issue with more complexity. I feel like the "unstyle" concept would be cleaner.

**Comment by lveillard** [comment]
Hey @ijjk! Thanks for reading the feature request. 
Did I do something wrong with the feature request? I followed the template 🤔

**Reply by ijjk** [reply]
Nothing wrong, we've been migrating feature requests to discussions under the ideas category. We've also been opening RFCs in discussions as well now. 

Note: there's also a few discussions talking about next/image styling. We're also looking at updating the documentation for `next/image`.

**Reply by lack-of-gravitas** [reply]
joke this discussion is still open and documentation is still lacking in 2022

**Comment by galago** [comment]
Another important thing is, that if there's a need for wrappers, better use span as it's more flexible than div; for instance, a div can't be inside a paragraph. With span that would be valid HTML and still be able to control the wrapper applying tailwind classes or with regular CSS

**Reply by lveillard** [reply]
I think they did it for nextjs 12

**Comment by jcamato** [comment]
I have also found Next Image component to be much more difficult to work with than the default img tag, especially when using Tailwind CSS.

**Comment by lveillard** [comment]
Next 12 is supposed to have <span> instead of <divs> which is already way better. But unfortunatelly i'm still unable to make it work with tailwind :S

**Comment by Amar-Gill** [comment]
I'm using nextjs 12 and I am unable to add borders with tailwindcss: `className='border-4 border-red-400 border-solid`. The `border:none` style on the `<img>` element is overriding it.

It works if I use the important override available in tailwindcss `className'!border-2 !border-red-500 !border-solid` but it would be nice if I didn't have to do this.
